package.json

```diff
@@ -12,7 +12,7 @@
     "format": "prettier --config prettier.config.js -w .",
     "format:check": "prettier --config prettier.config.js -c .",
     "lint": "eslint 'packages/*/src/**/*.{tsx,ts}'",
-    "test": "jest",
+    "test": "jest --testPathIgnorePatterns='packages/sdk/src/VisitorIdentifier.test.ts' && jest packages/sdk/src/VisitorIdentifier.test.ts --env=jsdom",
```

Comment:
Couldn't achieve this with test annotations or new configurations in `package.json`. This CLI-level configuration works, but it's not ideal

Reply:
So this is for allowing some type of tests to run in the `jsdom` environment?
Could we perhaps define an actual pattern for this? 
something like 
```
"test": "jest --testPathIgnorePatterns='packages/sdk/src/*.jsdom.test.ts' && jest packages/sdk/src/*.jsdom.test.ts --env=jsdom",
```
or however that would work?

Reply:
Really glad you took the time to figure out how to write these tests though! 👍 💯